.github/workflows/upload-framework.yml

```diff
@@ -30,7 +30,7 @@ jobs:
 
   upload-xcframework:
     needs: create-xcframework
-    runs-on: ubuntu-latest
+    runs-on: macos-latest
```

Comment:
Uploading xcframework.zip [fails](https://github.com/bucketeer-io/ios-client-sdk/actions/runs/8386420464/job/22967059352).

```
Post "https://uploads.github.com/repos/bucketeer-io/ios-client-sdk/releases/147852695/assets?label=&name=Bucketeer.xcframework.zip"
: read ./FrameworkBuild/Bucketeer.xcframework.zip: is a directory
```
I tried using the gh command locally, and even if I included the path in the file specification, I was able to upload it without any problems.
The point is that the Zip file is created in a Mac environment.
The `create-xcframework` job runs in a Mac OS, but `upload-xcframework` runs in an ubuntu OS, so I was wondering if the Zip file format might not be recognized properly.
We hope that the problem will be resolved by unifying the OS environment for Zip file generation and uploading jobs.

Reply:
Was this working using `ubuntu-latest`?
If so, the issue could be something else.